immut/sorted_map/sorted_map.mbti

```diff
@@ -20,6 +20,8 @@ fn filter[K : Compare, V](T[K, V], (V) -> Bool) -> T[K, V]
 
 fn filter_with_key[K : Compare, V](T[K, V], (K, V) -> Bool) -> T[K, V]
 
+fn find[K : Compare, V](T[K, V], K) -> V?
```

Comment:
We'd like to sticky on `lookup` instead of `find`. See `@array.T::find` which takes a predicate function instead of a key.